DotNET/Complex Flow Examples/ocr-with-extract-text.cs

```diff
@@ -0,0 +1,62 @@
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+class Program
+{
+    private static readonly string apiKey = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"; // Your API key here
+
+    static async Task Main(string[] args)
+    {
+        using (var httpClient = new HttpClient { BaseAddress = new Uri("https://api.pdfrest.com") })
+        {
+            // Upload PDF for OCR
+            using var ocrRequest = new HttpRequestMessage(HttpMethod.Post, "pdf-with-ocr-text");
+
+            ocrRequest.Headers.TryAddWithoutValidation("Api-Key", apiKey);
+            ocrRequest.Headers.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
+            var ocrMultipartContent = new MultipartFormDataContent();
+
+            var pdfByteArray = File.ReadAllBytes("/path/to/file.pdf");
+            var pdfByteArrayContent = new ByteArrayContent(pdfByteArray);
+            ocrMultipartContent.Add(pdfByteArrayContent, "file", "file.pdf");
+            pdfByteArrayContent.Headers.TryAddWithoutValidation("Content-Type", "application/pdf");
+            ocrMultipartContent.Add(new StringContent("example_pdf-with-ocr-text_out"), "output");
```

Comment:
We don't need to name the intermediate file. (Also, this line is causing trouble for the program -- see my comment below.) This line can be removed.